configuration/components/minio.libsonnet

```diff
@@ -5,6 +5,9 @@ local defaults = {
   local defaults = self,
   namespace: error 'must provide namespace',
 
+  buckets: ['thanos', 'loki'],
```

Comment:
Should we do the same as above here? Make these values empty and make them required?

Reply:
Yeah, makes sense, I'll do the same for access and secret key.

Reply:
done! :)